src/components/item/Container.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import { Item } from './Presenter';
+import axios from 'axios';
+
+export default class ItemContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      category: "",
+      description: "",
+      endDate: "",
+      id: props.match.params.id,
+      image: "",
+      loading: true,
+      price: "",
+      title: "",
+    }
+  }
+
+  fetchItem = async () => {
+    const self = this;
```

Comment:
Wondering why you are saving a reference to `this`?
the big arrow should give correct access to `this` under this function scope